src/common/Header/Input/index.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export default styled.input`
+  padding: 23px;
+  width: 432px;
+  // height: 48px;
```

Comment:
i wouldn't be leaving this type of comments. perhaps deleting is a better option